airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/JsonSchemaPrimitive.java

```diff
@@ -4,11 +4,28 @@
 
 package io.airbyte.protocol.models;
 
+import com.google.common.collect.ImmutableMap;
+
 public enum JsonSchemaPrimitive {
```

Comment:
might make sense to call this `JsonSchemaType` as these no longer represent primitives